x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/rule_bulk_actions/trial_license_complete_tier/perform_bulk_action.ts

```diff
@@ -1624,7 +1624,7 @@ export default ({ getService }: FtrProviderContext): void => {
                   {
                     type: BulkActionEditTypeEnum.set_rule_actions,
                     value: {
-                      throttle: '1h',
+                      throttle: '1d',
```

Comment:
Is there a preference here for mixing `'1d'` values with `'24h'` values? Should we standardize on one vs the other?

Reply:
I used `24h` everywhere, but here it had to be `1d`, because throttle has to be on of these
```ts

```